### ISSUE TITLE
Fluent UI TagPicker: Screen reader does not read the status as tag removed after tag removal

### ISSUE THREAD

**Issue by harshika-kashyap** [microsoft/fluentui#18616]
### Environment Information

- **Package version(s)**: 7.126.0
- **Browser and OS versions**: Chrome/Edge Windows10
- **Screen reader**: NVDA/Narrator

### Please provide a reproduction of the issue in a codepen:

Codepen link: https://codepen.io/harshika123/pen/gOmERZm

![image](https://user-images.githubusercontent.com/26279056/122527696-32449200-d039-11eb-94f4-53ce26123981.png)

#### Actual behavior:

Screen reader doesn't announce anything after tag removal.

#### Expected behavior:

Screen reader should read the status as "<tag-name> removed" after the tags are removed from the TagPicker.

​#### User Impact:​

Screen reader user will not get information regarding the tags removal from the field.

### Documentation describing expected behavior

https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

**Comment by gouttierre** [comment]
@harshika-kashyap - Thanks for filing this issue with us and providing repro steps. 

@smhigley - Let's discuss this ones priority level taking into consideration the lower version of fluent and also we will be replacing the website

**Comment by ecraig12345** [comment]
@gouttierre Although the repro includes a screenshot from the website, this isn't a website-specific issue. 

It appears that @smhigley's recent work has fixed this in version 8. At this point we probably won't be able to prioritize fixing it in version 7 unless it's critical to a product (though if someone else wants to port the fix back, we'd be happy to review it).

**Comment by smhigley** [comment]
@harshika-kashyap if you're interested in getting this into v7, the PR that fixed this in v8 is here: #19080. You could branch from v7, cherry-pick the commit, and open a PR for v7 if you'd up for it.

**Comment by mehulnirala** [comment]
#16097
